.github/workflows/bob.yml

```diff
@@ -0,0 +1,62 @@
+name: Bob build
+on:
```

Comment:
create a `github action` to publish ts-gen in `npm` for the `frontend` and `JS-SDK` (use a consistance package version)